kotlin/samples/tictactoe/common/src/main/java/com/squareup/sample/mainworkflow/MainWorkflow.kt

```diff
@@ -71,6 +71,8 @@ class MainWorkflow(
       val authScreen = context.renderChild(authWorkflow) { handleAuthResult(it) }
       val emptyGameScreen = GamePlayScreen()
 
+      // IDE is wrong, removing them breaks the compile.
```

Comment:
This looks like a bug in the new type inference in Kotlin plugin. Looks similar to [KT-32869](https://youtrack.jetbrains.com/issue/KT-32869).